### PR TITLE
Wait for devices to become available more reliably

### PR DESCRIPTION
**Pull request by a1exsh**
- Increase default wait timeout to 60 seconds.
- Exit with error code if the device doesn't appear after timeout.

### CODE REVIEW

**Comment by a1exsh** [comment]
This should fix #322

File: runtime/opt/taupage/init.d/10-prepare-disks.py [hunk 1]
```diff
@@ -57,13 +57,18 @@ def attach_volume(ec2, volume_id, attach_as):
...
-    while tries < max_tries and not os.path.exists(device):
+    while tries < max_tries:
+        if os.path.exists(device):
+            return
         logging.info("Waiting for %s to stabilize..", device)
         tries += 1
         sleep(wait_time)
+    logging.error("Failed to wait for %s device to become available after %s seconds",
+                  device, max_tries * wait_time)
+    sys.exit(2)
```
Comment (Raffo) [inline runtime/opt/taupage/init.d/10-prepare-disks.py]: @a1exsh what is the reason of this exit?

Comment (a1exsh) [inline runtime/opt/taupage/init.d/10-prepare-disks.py]: The reason is: https://github.com/zalando/planb-cassandra/issues/93#issuecomment-263242986
> The problem seems to be that Taupage's wait_for_device by default only waits for up to 15 seconds. If you are unlucky to attach the volume just after mount failure, the next step is to format it.

Basically, it is safer to stop the boot up procedure than risk formatting some really big EBS with some really important data ;-)

**Comment by slitsche** [comment]
👍

File: runtime/opt/taupage/init.d/10-prepare-disks.py (hunk 1 above)
Comment (Raffo) [inline runtime/opt/taupage/init.d/10-prepare-disks.py]: Sounds reasonable.

**Comment by Raffo** [comment]
👍

**Comment by Raffo** [comment]
@a1exsh tests are broken, is this related to your changes?

**Comment by a1exsh** [comment]
@Raffo not at all.  flake8 is complaining about files the commit didn't touch:
```

./runtime/opt/taupage/bin/decrypt-kms.py:30:1: E305 expected 2 blank lines after class or function definition, found 1
./runtime/opt/taupage/bin/register-in-etcd.py:93:1: E305 expected 2 blank lines after class or function definition, found 1
./runtime/opt/taupage/init.d/03-push-taupage-yaml.py:58:1: E305 expected 2 blank lines after class or function definition, found 1
./runtime/opt/taupage/init.d/04-configure-cloudwatch-logs-agent.py:111:1: E305 expected 2 blank lines after class or function definition, found 1
```

**Comment by a1exsh** [comment]
@Raffo any ETA on this? Thank you.

**Comment by hjacobs** [comment]
@a1exsh please just fix the flake8 issues, these are new issues because they were not checked in old flake8 versions

**Comment by Raffo** [comment]
After @hjacobs comment is resolved, we will approve and schedule a Taupage release.

**Comment by Raffo** [comment]
@a1exsh Will fix the flake8 issues, then ask for rebase.

**Comment by Raffo** [comment]
@a1exsh Tests should pass now in master.

**Comment by Raffo** [comment]
For reference: https://github.com/zalando-stups/taupage/pull/325/files

**Comment by mikkeloscar** [comment]
@Raffo restarted the tests, they are now passing.

**Comment by Raffo** [comment]
👍

**Pull request merged**